docs/build/build-integrate-assets.md

```diff
@@ -20,8 +20,8 @@ Beyond merely supporting assets, integrating an Asset Hub into your systems has
 infrastructure providers and users:
 
 - Support for on-chain assets.
-- Significantly lower transaction fees (about 1/10) than the Relay Chain.
-- Significantly lower deposits (1/10) than the Relay Chain. This includes the existential deposit
+- Significantly lower transaction fees (about 1/100) than the Relay Chain.
```

Comment:
The transaction fees is still about 1/10th, although the deposits were lowered to 1/100
cc: @joepetrowski 

```suggestion
- Significantly lower transaction fees (about 1/10) than the Relay Chain.
```